src/main/kotlin/net/ccbluex/liquidbounce/event/EventListener.kt

```diff
@@ -101,11 +101,25 @@ inline fun <reified T : Event> EventListener.until(
 inline fun <reified T : Event> EventListener.once(
     priority: Short = 0,
     crossinline handler: (T) -> Unit
-): EventHook<T> = until(priority) { event ->
+): EventHook<T> = until(priority) { event -> // Don't use `repated` 'cause for no overhead
```

Comment:
```suggestion
): EventHook<T> = until(priority) { event -> // Don't use `repeated` 'cause for no overhead
```
:nerd_face::point_up: